### ISSUE TITLE
Want to add an arbitrary span attribute.

### ISSUE THREAD

**Issue by yuzujoe** [inception-health/otel-export-trace-action#193]
Currently I can pass in any value for the Trace Unique Field, but can I set any other value as a key?

https://github.com/inception-health/otel-export-trace-action#trace-unique-fields

I would like to assign a value for observability.

**Comment by utezduyar** [comment]
We also have a similar use case. Our opentelemetry endpoint requires team resource attribute on the spans. Without this attribute spans are filtered out. Is there a way to add custom attributes?

**Comment by Pixie-Axerup** [comment]
Same here, to be able to send to our endpoint we need to add resource attributes.

**Comment by Amit-Baranes-Deel** [comment]
any update here?

**Comment by corentinmusard** [comment]
If you still need this, I just added this feature on my [fork](https://github.com/corentinmusard/otel-cicd-action?tab=readme-ov-file#adding-arbitrary-resource-attributes).

**Comment by utezduyar** [comment]
Will you try to merge it in here?

**Comment by corentinmusard** [comment]
No, this repo has no activity since more than 2 years. People already tried to merge fixes here without success https://github.com/inception-health/otel-export-trace-action/pull/197.

I actually fixed all the currently opened issues and PRs in my fork.
